pkgs/by-name/te/terminal-toys/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: rec {
```

There's no consensus or rule about using finalAttrs vs rec, you can use what ever you prefer if no specific need. You can get more context here https://github.com/NixOS/nixpkgs/issues/315337

But if you are going to use `finalAttrs`, `rec` is not required here anymore
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```